src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -246,9 +246,16 @@ def translate_as_fieldop(
     stencil_expr, domain_expr = fun_node.args
 
     if isinstance(stencil_expr, gtir.Lambda):
-        # Default case, handled below: the argument expression is a lambda function
-        # representing the stencil operation to be computed over the field domain.
-        pass
+        if cpm.is_call_to(stencil_expr.expr, "deref") and cpm.is_ref_to(
+            stencil_expr.expr.args[0], stencil_expr.params[0].id
+        ):
+            # Prune trivial field operators with the form '(⇑(λ(__arg) → ·__arg))(a)'
+            # that are doing element-wise copy of the input field.
```

Comment:
As far as I understood our discussion, this will translate the Map that does the copy into a Memlet?
I would add to the description above to what this case is translated to.